code/game/objects/items/defib.dm

```diff
@@ -80,6 +80,10 @@
 /obj/item/defibrillator/ui_action_click()
 	toggle_paddles()
 
+/obj/item/defibrillator/CtrlClick()
+	if(ishuman(usr) && Adjacent(usr))
```

Comment:
Can't wait to pick up the paddles while sleeping. That's gonna be lit :100:

Reply:
yes 
![image](https://user-images.githubusercontent.com/30060146/60228464-c4599b80-9860-11e9-9f17-da336d78d695.png)

Reply:
Still a bit odd that you even get the message, don't you think? Just use canusetopic.